source/core/parameters.cc

```diff
@@ -1030,6 +1055,11 @@ namespace Parameters
                         "1",
                         Patterns::Integer(),
                         "amg smoother overlap");
+      prm.declare_entry(
+        "block linear solver fail",
+        "false",
+        Patterns::Bool(),
+        "A bool that allow the use to block linear solver to throw errors");
```

Comment:
You should find a better name for this parameter. Like force linear solver continuation?
Something like that. Try to add a better description of it also :).